Fonts/Scripts/download-relative-fonts

```diff
@@ -9,8 +9,10 @@ git init
 git remote add origin git@github.skyscannertools.net:backpack/bpk-fonts.git
 git config core.sparsecheckout true
 echo "iOS" >> .git/info/sparse-checkout
-git pull --depth=1 origin main
+# We should be smarter about this, as it currently takes too long to fail.
+# Maybe we should only pull fonts if not already present, as most internal devs will have them from a previous install
+git pull --depth=1 origin main || echo "\033[0;31mFailed to pull internal Skyscanner fonts. If you want to develop with the fonts, check that you are on the VPN\033[0m"
```

Comment:
For now I've just updated the existing font pulling script.
Need to make this a bit smarter and probably rewrite in ruby